---
ram_msgs/AdditiveManufacturingTrajectoryInfo informations
